abaverify/main.py

```diff
@@ -452,7 +452,21 @@ def _runAssertionsOnResults(self, jobName):
                 # Loop through values if there are more than one
                 if hasattr(r['computedValue'], '__iter__'):
                     for i in range(0, len(r['computedValue'])):
-                        self.assertAlmostEqual(r['computedValue'][i], r['referenceValue'][i], delta=r['tolerance'][i])
+                        computed_val = r['computedValue'][i]
+                        reference_val = r['referenceValue'][i]
+                        delta = r['tolerance'][i]
+                        if isinstance(computed_val, tuple):
```

Comment:
Does it make more sense to check the reference value since the computed value could be garbage in the case that the test failed?